codemods/tiptap-2-migrate-imports/src/index.ts

```diff
@@ -27,7 +27,7 @@ export default function transform(file: any, api: any) {
     "@tiptap/extension-dropcursor": "Dropcursor",
     "@tiptap/extension-gapcursor": "Gapcursor",
     "@tiptap/extension-focus": "Focus",
-    "@tiptap/extension-history": "History",
+    "@tiptap/extension-undo-redo": "UndoRedo",
```

Comment:
This would rename a package they are pretty much guaranteed to not have already, so effectively doing nothing

Reply:
Thanks, reversed the change.